source/documentation/managing_apps/quotas.md

```diff
@@ -20,7 +20,31 @@ To see your organisation quota, run the command:
 
 ``cf org YOURORG``
 
-where YOURORG is your organisation's name. (If you don't know the name, you can use ``cf orgs`` to find out).
+where YOURORG is your organisation's name. (If you don't know the name, you can use ``cf orgs`` to find out). This will produce an output which includes the name of the organisation's current quota:
+
+```
+name:                  YOURORG
+domains:               my.domain,
+**quota:                 medium**
+spaces:                space1, space2
+isolation segments:
+```
+
+To see the details of that quota, run:
+
+``cf quota QUOTA``
+
+where QUOTA is the name of the quota your organisation has been assigned.
```

Comment:
"...where `QUOTA` is the name of the quota that GOV.UK PaaS has assigned your org."